docs/rpc@usage/nameserver.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+import Pyro4
+import json
+
+name_server_address = '192.168.122.34'
+#Starting the NameServer
```

Comment:
Again, can this parameter be passed from the script which is calling this python function.